articles/quickstart/webapp/java-ee/_includes/_login.md

```diff
@@ -119,7 +119,16 @@ public class Auth0JwtIdentityStore implements IdentityStore {
 
 If the `credential` is an `Auth0Credential`, the calling user is authenticated and valid, so a `CredentialValidationResult` created with the credential is returned to indicate success. If it is not an `Auth0Credential`, return `CredentialValidationResult.NOT_VALIDATED_RESULT`.
 
-Before implementing the `HttpAuthenticationMechanism` interface that will use all these collaborators, create a bean that will provide a configured instance of the `AuthenticationController` from the Auth0 Java MVC SDK, used to build the authorize URLs and handle the token exchange to authenticate users:
+Before implementing the `HttpAuthenticationMechanism` interface that will use all these collaborators, create a bean that will provide a configured instance of the `AuthenticationController` from the Auth0 Java MVC SDK. The `AuthenticationController` is used to build the authorize URL where users will login, and handle the token exchange to authenticate users. 
```

Comment:
I think you mean "authentication URL" here?

Reply:
Actually it should be "authorization url" (https://auth0.com/docs/flows/guides/auth-code/add-login-auth-code#example-authorization-url). I'll update it.